spec/src/main/asciidoc/jakarta-stl.adoc

```diff
@@ -659,8 +658,8 @@ Date: <c:out value="${formattedDate}" default="not specified"/>
 
 == Expression Language Overview
 
-JSTL 1.0 introduced the notion of an
-expression language (EL) to make it easy for page authors to access and
+Jakarta Standard Tag Library 1.0 introduced the notion of an
+expression language to make it easy for page authors to access and
```

Comment:
We really can't use Jakarta Standard Tag Library here can we ? I say this because there was no Jakarta Standard Tag Library version 1.0. Should we reword this using the JSR spec number and version perhaps?

Reply:
Reading this again, I think we'd better remove the entire historical reference. Maybe it once made sense when EL was just moved, but I doesn't 16 years later.

Also, the text is factual wrong I realise, since the expresion language has again been moved out to its own API. It's not in Jakarta Server Pages.

Perhaps best to just introduce expression language as what it currently does.